examples/vision/image_classification_efficientnet_fine_tuning.py

```diff
@@ -0,0 +1,423 @@
+"""
+Title: Image classification using EfficientNet and fine-tuning
+Author: Yixing Fu
+Date created: 2020/06/30
+Last modified: 2020/07/06
+Description: Use EfficientNet with weights pre-trained on imagenet for CIFAR-100 classification.
+"""
+"""
+
+## What is EfficientNet
+EfficientNet, first introduced in https://arxiv.org/abs/1905.11946 is among the most
```

Comment:
Please use markdown formatting for links, e.g. 

`[Tan and Le, 2019](https://arxiv.org/abs/1905.11946)`

Applicable to all links in this file.